astro/src/content/docs/lifecycle/authenticate-users/single-sign-on.mdx

```diff
@@ -376,5 +376,5 @@ Navigate to <Breadcrumb>Applications -> Your Application -> OAuth</Breadcrumb> a
 
 * You can view the [example application's codebase](https://github.com/fusionauth/fusionauth-example-node-sso).
 * The [Tenant API](/docs/apis/tenants) can be used to manage single sign-on related configuration.
-* This guide uses the hosted login pages. If you are using the Login API and building your own pages, [check out the comments on this issue](https://github.com/FusionAuth/fusionauth-issues/issues/171) for guidance.
+* This guide uses the hosted login pages. If you are using the Login API and building your own pages, [see our Login API Overview](/docs/lifecycle/authenticate-users/login-api/) for guidance.
```

Comment:
Hmmm. If you look at the comments on 171, particularly https://github.com/FusionAuth/fusionauth-issues/issues/171#issuecomment-495802205 then it is clear our overview is not a substitute. 

But I don't think we want to document using the login API to recreate SSO at this time (which would require sifting through those comments, making an example app, etc).

Reply:
```suggestion
* This guide uses the hosted login pages.
```

So I'd suggest this.